### DISCUSSION TITLE
Change breathing effect to Progress

### DISCUSSION THREAD

**Discussion by Scooby7274** [julianschill/klipper-led_effect#214] [Q&A] [answered]
Uisng the Simulator and when I change the effect from "breathing" to "progress",it doesn't work. Trying to get my 15 led Neopixel to show temp progress for my extruder,etc.

**Comment by julianschill** [comment]
Progress shows the print progress. (How many percent of your print is already done). Use the progress slider in the Simulator. To show the temperatures use the HeaterGauge or TemperatureGauge effect.

**Reply by Scooby7274** [reply]
I'm getting a "heater not defined error" on both. I'll have to see what config I'm missing.

**Reply by julianschill** [reply, marked as answer]
You have to define on which heater the effect should react on. E.g
```heater: extruder```
Or
```heater: "temperature_fan myfan"```

Check the

**Reply by Scooby7274** [reply]
Thanks julian,I will mess with it more tomorrow.
